src/modules/image.cxx

```diff
@@ -0,0 +1,39 @@
+module;
+
+#include <cstdint>
+#include <vector>
+#include <iostream>
+
+export module Image;
+
+export class Image
+{
+    const int channels, width, height, max_value;
+    std::vector<uint8_t> pixels;
```

Comment:
[remark] See why I said that the image library should expose the pixel type? Internally you are using `uint8_t`, in main you're using `int` to represent pixels.